### DISCUSSION TITLE
开启embedded，从dnd面板拖动图形到画布，会触发added和embedded两个事件

### DISCUSSION THREAD

**Discussion by hysma** [antvis/X6#2226] [Q&A]
如标题描述，此时未做嵌入操作，仍然会触发embedded事件，麻烦问下该现象是否正常呢？
例子如下
https://codesandbox.io/s/red-meadow-8xscqc?file=/src/app.tsx

**Comment by NewByVector** [comment]
这应该是个 bug

**Reply by NewByVector** [reply]
https://github.com/antvis/X6/issues/2240
